src/app/screens/Search/index.tsx

```diff
@@ -15,12 +19,22 @@ function Search() {
 
   return (
     <View style={styles.mainContainer}>
-      <FlatList
-        data={foundedBooks}
-        renderItem={renderItem}
-        keyExtractor={setId}
-        ItemSeparatorComponent={renderSeparator}
-      />
+      {foundedBooks.length != 0 ? 
```

Comment:
`foundedBooks` sounds weird, why don't you rename it to `searchBooks`, also you can check like this 
```js
searchBooks.length ?